sentry_sdk/serializer.py

```diff
@@ -182,10 +188,11 @@ def _is_databag():
             if rv in (True, None):
                 return rv
 
-            p0 = path[0]
-            if p0 == "request" and path[1] == "data":
-                return True
+            is_request_body = _is_request_body()
+            if is_request_body in (True, None):
```

Comment:
I do not get why there is now a "None" case? Can you explain this to me"

Reply:
`None` basically means we're currently recursively looking at a data level where there is no `['request']['data']`, but it's still possible that we'll encounter it somewhere further down the tree (as opposed to `False` which means we're somewhere where one can't expect a request body to appear anymore). The behavior basically copies what we do [here](https://github.com/getsentry/sentry-python/blob/master/sentry_sdk/serializer.py#L148-L152) and [here](https://github.com/getsentry/sentry-python/blob/master/sentry_sdk/serializer.py#L177-L178).

Reply:
oh, now I get it. Thanks for explaining!